expression/builtin_arithmetic.go

```diff
@@ -594,6 +603,7 @@ func (c *arithmeticIntDivideFunctionClass) getFunction(ctx context.Context, args
 }
 
 type builtinArithmeticIntDivideIntSig struct{ baseBuiltinFunc }
+type builtinArithmeticRealDivideRealSig struct{ baseBuiltinFunc }
```

Comment:
I think the `IntDivide` represents the 'DIV` operator, so the name should be `IntDivideRealSig`.

Reply:
You are right. PTAL.